src/main/java/io/confluent/kafka/connect/datagen/DatagenTask.java

```diff
@@ -81,7 +81,8 @@ protected enum Quickstart {
     PAGEVIEWS("pageviews_schema.avro", "viewtime"),
     STOCK_TRADES("stock_trades_schema.avro", "symbol"),
     INVENTORY("inventory.avro", "id"),
-    PRODUCT("product.avro", "id");
+    PRODUCT("product.avro", "id"),
+    PERSON("person_schema.avro", "person");
```

Comment:
```suggestion
    PERSON("person_schema.avro", "person");
```
Please remove this from the quick start.